src/proteus/utils/logs.py

```diff
@@ -57,19 +57,22 @@ def format(self, record):
         return formatter.format(record)
 
 # Custom logger instance
-def SetupLogger(logpath:str="new.log",level:str="INFO",logterm:bool=True):
+def setup_logger(logpath:str="new.log",level:str="INFO",logterm:bool=True):
+
+    logger_name = "fwl"
 
     # https://stackoverflow.com/a/61457119
+    print("Setup logger as %s"%str(logger_name))
```

Comment:
Is this a debug statment that you meant to remove?

```suggestion
    print(f"Setup logger as {logger_name}")
```